### PR TITLE
fixed the issue that external etcd is not ready

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Reference: https://github.com/open-cluster-management-io/multicluster-controlplane/pull/33
Error message:
```bash
Checking multicluster-etcd...
Error:  client: etcd cluster is unavailable or misconfigured; error #0: dial tcp 127.0.0.1:4001: getsockopt: connection refused
; error #1: dial tcp 127.0.0.1:2379: getsockopt: connection refused

error #0: dial tcp 127.0.0.1:4001: getsockopt: connection refused
error #1: dial tcp 127.0.0.1:2379: getsockopt: connection refused

command terminated with exit code 4
!!!!!!!!!!  the multicluster-etcd multicluster-controlplane-etcd is not ready within 60s
```

### CODE REVIEW

File: hack/deploy-etcd.sh [hunk 1]
```diff
@@ -62,16 +62,16 @@ ${KUSTOMIZE} build ${KUBE_ROOT}/hack/deploy/etcd | ${KUBECTL} apply -f -
...
         RESULT=$(${KUBECTL} -n ${ETCD_NS} exec etcd-0 -- etcdctl cluster-health | tail -n1)
-        if [  "${RESULT}" = "cluster is healthy" ]; then
+        if [[ "${RESULT}" = "cluster is healthy" ]]; then
             echo "#### multicluster-etcd ${ETCD_NS} is ready ####"
             break
         fi
         
-        if [ $i -eq 20 ]; then
-            echo "!!!!!!!!!!  the multicluster-etcd ${ETCD_NS} is not ready within 60s"
+        if [ $i -eq 30 ]; then
```
Comment (clyang82) [inline hack/deploy-etcd.sh:73]: 30 or 50?

Comment (yanmxa) [reply hack/deploy-etcd.sh]: It should be 50.
Done

**Pull request merged**